src/tenant-token/seed/package.json

```diff
@@ -5,13 +5,13 @@
   "main": "index.js",
   "scripts": {
     "start": "node index.js",
-    "meilisearch-serve": "MEILI_API_KEY=$(grep MEILI_API_KEY .env | cut -d '=' -f2) && docker run -it --rm -p 7700:7700 -v $(pwd)/data.ms:/data.ms getmeili/meilisearch:latest ./meilisearch --master-key=$MEILI_API_KEY",
+    "meilisearch-serve": "MEILI_MASTER_KEY=$(grep MEILI_MASTER_KEY .env | cut -d '=' -f2) && docker run -it --rm -p 7700:7700 -v $(pwd)/meili_data:/meili_data getmeili/meilisearch:v0.30.4 meilisearch --master-key=$MEILI_MASTER_KEY",
```

Comment:
```suggestion
    "meilisearch-serve": "MEILI_MASTER_KEY=$(grep MEILI_MASTER_KEY .env | cut -d '=' -f2 | tr -d \"'\" | tr -d '\"') && docker run -it --rm -p 7700:7700 -v $(pwd)/meili_data:/meili_data getmeili/meilisearch:v0.30.4 meilisearch --master-key=$MEILI_MASTER_KEY",
```

I added this part to get rid of quotes. Usually, I add my env vars as strings. I noticed that when I tried to add the documents by running `npm start` it threw an invalid api key error because I had launched the meilisearch instance with `"masterKey"` instead of `masterKey`.